rplugin/python3/iron/repls/php.py

```diff
@@ -0,0 +1,15 @@
+# encoding:utf-8
+"""PHP repl definition for iron.nvim. """
+from iron.repls.utils.cmd import detect_fn
+
+php = {
+    'command': 'php -a',
+    'language': 'php',
+    'detect': detect_fn('php'),
+}
+
+psyshell = {
+    'command': 'psysh',
+    'language': 'php',
+    'detect': detect_fn('php'),
```

Comment:
This should look for the `psysh`bibary so it'd be a preferred option over the `php` binary.